supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -0,0 +1,157 @@
+package cromwell.backend.impl.jes
```

Comment:
OK, from your comment I'm deducing you are OK with the change. So I will make the change.